services/comments/comments.go

```diff
@@ -42,11 +42,26 @@ func CreateIssueComment(doer *models.User, repo *models.Repository, issue *model
 
 // UpdateComment updates information of comment.
 func UpdateComment(c *models.Comment, doer *models.User, oldContent string) error {
+	var needsContentHistory = c.Content != oldContent &&
+		(c.Type == models.CommentTypeComment || c.Type == models.CommentTypeReview || c.Type == models.CommentTypeCode)
```

Comment:
A little curious, what is a `CommentTypeCode` comment (I haven't read the PR code yet)

Reply:
CommentTypeCode is a comment on a line of code. CommentTypeReview is the main summarizing comment you can add when submitting the entire review (which can consist of multiple CommentTypeCode).